dbt/config.py

```diff
@@ -25,14 +25,8 @@ def read_profile(profiles_dir):
 
 def read_config(profiles_dir):
     profile = read_profile(profiles_dir)
-    return profile.get('config')
+    return profile.get('config', {})
 
 
-def send_anonymous_usage_stats(profiles_dir):
-    config = read_config(profiles_dir)
-
-    if config is not None \
-       and not config.get("send_anonymous_usage_stats", True):
-        return False
-
-    return True
+def send_anonymous_usage_stats(config):
+    return config.get('send_anonymous_usage_stats', True)
```

Comment:
@drewbanin I'm not sure this works. There are some unit test failures in circle & appveyor